test/test_integration_policy.py

```diff
@@ -0,0 +1,316 @@
+# -*- coding: utf-8 -*-
+
+"""
+CLI Integration tests
+
+This test file handles the main test flows after initialization and configuration
+"""
+import io
+import json
+import tempfile
+import uuid
+from contextlib import redirect_stderr
+
+import Utils
+from conjur.constants import *
+from .util.cli_helpers import integration_test
+from test.util.test_runners.integration_test_case import IntegrationTestCaseBase
+
+
+# Not coverage tested since integration tests doesn't run in
+# the same build step
+class CliIntegrationPolicy(IntegrationTestCaseBase):  # pragma: no cover
+    capture_stream = io.StringIO()
+    def __init__(self, testname, client_params=None, environment_params=None):
+        super(CliIntegrationPolicy, self).__init__(testname, client_params, environment_params)
+
+    # *************** HELPERS ***************
+
+    def setup_cli_params(self, env_vars, *params):
+        self.cli_auth_params = ['--debug']
+        self.cli_auth_params += params
+
+        return self.cli_auth_params
+
+    def setUp(self):
+        self.setup_cli_params({})
+        Utils.setup_cli(self)
+        return self.invoke_cli(self.cli_auth_params,
+                               ['policy', 'replace', '-b', 'root', '-f', self.environment.path_provider.get_policy_path("initial")])
+
+    # *************** TESTS ***************
+
+    @integration_test
+    def test_https_can_load_policy(self):
+        self.setup_cli_params({})
+
+        policy, variables = Utils.generate_policy_string(self)
+        Utils.load_policy_from_string(self, policy)
+
+        for variable in variables:
+            Utils.assert_set_and_get(self, variable)
+
+    @integration_test
+    def test_https_load_policy_can_output_returned_data(self):
```

Comment:
The name is not so clear since in general we always get output when we load policy (even if we get empty string) we still get the json_result version

Reply:
updating to `test_load_policy_of_new_resources_returns_new_entry_json_data` to be clear that this is the json for creating new entries. WDYT?